Dockerfile.composer

```diff
@@ -0,0 +1,24 @@
+FROM composer:1.6.4
+
+ARG composer_dev_arg
+
+RUN apk add --no-cache \
+    patch
+
+COPY composer.json \
+    composer.lock \
+    ./
+
+RUN composer --no-interaction install ${composer_dev_arg} --ignore-platform-reqs --no-autoloader --no-suggest
+
+COPY test/ test/
+COPY src/ src/
+
+RUN composer --no-interaction dump-autoload ${composer_dev_arg} --classmap-authoritative
+
+RUN mkdir web
+COPY app/ app/
+COPY .docker/parameters.yml app/config/
+COPY bin/ bin/
+
+RUN bin/console assets:install
```

Comment:
Bit annoying to do it here, either need to do multistage in the `assets` image or here (or use another intermediate image).